### ISSUE TITLE
ical single_event export not create an event

### ISSUE THREAD

**Issue by muescha** [hacken-in/website#555]
## where

- on single event page
- click on `iCal Download`
- select `open with: CalendarFileHandler (Standard)`
- Calendar app starts

## what i see
- it just open the calendar app
- no new event dialog

## what i expected
- an dialog to add the event to my calendar

## environment
- Firefox 35
- OSX 10.9.5

**Comment by bitboxer** [comment]
Can you say on what entry you tried this? Worked on all 4 I tried right now.

**Comment by bitboxer** [comment]
@muescha ping?

**Comment by bascht** [comment]
@muescha This looks like a problem with your client integration.
The file itself is valid:
![screenshot from 2015-02-05 08 45 05](https://cloud.githubusercontent.com/assets/51264/6056651/40cf66dc-ad14-11e4-99a3-7713ee1d2f61.png)

And we're sending the correct headers:

```http
> curl --head https://hacken.in/export/ical/single_event/5394
HTTP/1.1 200 OK
Server: nginx/1.7.7
Date: Thu, 05 Feb 2015 07:50:42 GMT
Content-Type: text/calendar; charset=UTF-8
Connection: keep-alive
Strict-Transport-Security: max-age=31536000
X-Frame-Options: SAMEORIGIN
X-XSS-Protection: 1; mode=block
X-Content-Type-Options: nosniff
ETag: "6571d9a3e5a1aade3aad972e7c73d950"
Set-Cookie: ahoy_visitor=fb9ef5cd-61e0-4f7e-80e5-570ec9b0f91e; path=/; expires=Sun, 05 Feb 2017 07:50:42 -0000; secure
Set-Cookie: ahoy_visit=cdad3555-8d7b-4f40-bb9e-7712de6fea43; path=/; expires=Thu, 05 Feb 2015 11:50:42 -0000; secure
X-Request-Id: ea312a3d-95a4-4665-a108-abd40f503f49
X-Runtime: 0.009237
Cache-Control: max-age=0, private, must-revalidate
X-Varnish: 627206
Age: 0
Via: 1.1 varnish-v4
X-Varnish-Cache: MISS

```

Can you cross-check using another .ical file or triggering the download with another browser / opening the file directly.

If you have a reproducible example for an .ical export that "works" for you so we can confront it to ours: Just reopen.

**Comment by muescha** [comment]
sorry for delay.

## tl;dr

with extension .ics it works in Firefox:
`https://hacken.in/export/ical/single_event/5412` --> not works
`https://hacken.in/export/ical/single_event/5412.ics` --> this works



## my cross checks:

### Safari 7.1.2
- click on ical opens Kalender and add the event

### Chrome
- click on ical opens only a save as `1234.ics` file - no other choices

### Firefox 35.0.1
- click on ical i get an:
```
Sie möchten folgende Datei öffnen: 
5412 
vom Typ: ICS file
von: https://hacken.in/
```
- i can choose: `Kalender` or `CalendarFileHandler (Standard)` --> nothing
- saved file `5412` - i can not open it with Kalender
- renamed file `5412.ics` i can open it with Kalender

### last try with Firefox 35.0.1
i added a .ics to the link `https://hacken.in/export/ical/single_event/5412.ics` now it works how expected

**Comment by bitboxer** [comment]
I think this is a misconfiguration on your system. I tried on 3 different macs, a windows and a linux machine. It works everywhere.

**Comment by bascht** [comment]
@bitboxer for the sake of having a clean ~/Downloads folder: We could add an ".ics" extension to the url, couldn't we?
